### PR TITLE
Update Sycamore Readme

### PR DESCRIPTION
**Pull request by clementzach**
Made Readme more complete, including a summary for each generated table

### CODE REVIEW

**Comment by clementzach** [comment]
@hackdna are there any guidelines about how to format the examples in README files? I followed the example of whoever wrote the README file, but as the examples are, they do not follow PEP 8, and code is not runnable.

**Review by hackdna — CHANGES_REQUESTED** [review]
There are no hard and fast rules about READMEs. The overall idea is to provide enough information to make it easy to use the module/package. Having said that, there are some examples you could follow:

https://docs.python.org/3/library/datetime.html#date-objects
https://docs.djangoproject.com/en/4.0/ref/files/file/#the-contentfile-class
https://docs.python-requests.org/en/latest/api/

Note that you don't have to provide code examples for all functions. A function signature with a description of arguments and return values is often sufficient because it is self-explanatory. When a code example is provided then it should have some actual variable values, not just a description of values. Also, ideally code examples should be very small, so line length won't be an issue (see documentation examples above).

Also, API reference can be generated dynamically from docstrings using Sphinx and hosted on Read The Docs (https://brendanhasz.github.io/2019/01/05/sphinx.html). This can help avoid copy/paste but would require some initial setup.

Finally, it would be great to think about restructuring the module to make it a bit easier to use. Consider this example: `forest.sycamore.sycamore_main.survey_stats_main()`
Words `sycamore` and `main` are both repeated twice, so it would be great to streamline the naming (here's a good discussion of module naming and structure: https://docs.python-guide.org/writing/structure/#modules).

I'd like to suggest the following changes:
```
survey_main.py -> base.py
functions.py -> common.py
changed_answers.py -> responses.py
survey_config.py -> survey.py or config.py

survey_stats_main() -> survey_stats()
```
Then we could add a `__main__.py` with a call to `base.survey_stats()` which would allow this usage:
```
python -m sycamore <args>
```
Details: https://docs.python.org/3/library/__main__.html#main-py-in-python-packages

And if we add to `__init__.py`:
```
from .base import survey_stats
from .common import get_survey_timings
etc
```
then we could do:
```
import sycamore
sycamore.survey_stats()
```
Discussion here: https://towardsdatascience.com/whats-init-for-me-d70a312da583

Thoughts?

**Comment by clementzach** [comment]
@hackdna thanks for your suggestions about __init__.py and __main__.py. I didn't know that you could do that, and I think making those changes will make the package will be a lot more user-friendly. 

I made changes to the module names as well, and I think that will clarify things a lot for users. 

I did have a question about the API reference created using sphinx. Are we hoping that this reference will replace the wiki we currently have, or will it be supplemental for those looking for specifics about function parameters? I created a rough landing page (at docs/_build/html/index.html) that seems like it would at least be useful for those looking for specifics about parameters in main functions. 

And, I believe that if we want to make an account on readthedocs.io, it should be with an email that will stay with the lab, correct?

And finally, I realized that the other modules do not have a README file in their folder, just a wiki entry. Should I just remove the README from sycamore's folder rather than duplicating what's on the wiki?

File: forest/sycamore/__main__.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import sys
+
+import argparse
```
[the remark this answers is not in the document]
Comment (clementzach) [reply forest/sycamore/__main__.py:1]: Ah, didn't realize this was built in

**Comment by clementzach** [comment]
@hackdna I reached out to the maintainer of the other forest package to see if he's still using it. 

While we wait for him to get back to us, and while we wait to discuss what we would want to do for documentation as a team, should I create a separate pull request that contains just the edits to the sycamore code from folder name changes?

File: docs/_build/html/_sources/index.rst.txt [hunk 2]
```diff
@@ -0,0 +1,55 @@
+.. forest documentation master file, created by
```
[the remark this answers is not in the document]
Comment (hackdna) [reply docs/_build/html/_sources/index.rst.txt:1]: The docs build dir is still part of the repo.

File: forest/sycamore/__main__.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (hackdna) [reply forest/sycamore/__main__.py:1]: Both packages are builtins so should be grouped together (https://peps.python.org/pep-0008/#imports)

**Review by hackdna — CHANGES_REQUESTED** [review]
Sure, please feel free to create a separate PR for the code restructuring changes.

**Review by hackdna — APPROVED** [review]

**Pull request merged**